### PR TITLE
Bring in new utils for new python requirements-freeze command

### PR DESCRIPTION
**Pull request by benvand**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,22 +6,12 @@ virtualenv:
...
-	rm -rf venv-freeze
+freeze-requirements: virtualenv requirements-dev requirements-app.txt
+	${VIRTUALENV_ROOT}/bin/python -m dmutils.repoutils.freeze_requirements requirements-app.txt
 
-.PHONY: test-requirements
-test-requirements:
-	@diff requirements-app.txt requirements.txt | grep '<' \
-	    && { echo "requirements.txt doesn't match requirements-app.txt."; \
-	         echo "Run 'make freeze-requirements' to update."; exit 1; } \
-	    || { echo "requirements.txt is up to date"; exit 0; }
```
Comment (katstevens) [inline Makefile:24]: Why are we getting rid of `test-requirements` here?

Comment (benvand) [reply Makefile:24]: Replaced this. Accidentally removed.

Comment (benvand) [reply Makefile:24]: Sorry, actually, I removed it because it's duplicated in that file.

Comment (benvand) [reply Makefile:24]: Hard to keep track 😬

Comment (katstevens) [reply Makefile:24]: Cool 😸Works locally for me

**Review by katstevens — APPROVED** [review]

**Pull request merged**